Source/UI/NewUI/CommonUI/View/TextFieldView.swift

```diff
@@ -81,6 +90,13 @@ class TextFieldView: UIView {
         headerLabel?.textColor = style.title?.textColor
     }
 
+    private func updateIsOptionalLabel(style: CellTextFieldStyle) {
+        isOptionalLabel?.isHidden = !style.isOptional
+        isOptionalLabel?.text = Constants.LocalizationKeys.BillingForm.Cell.isOptional
+        isOptionalLabel?.font = UIFont(graphikStyle: .regular, size:  Constants.Style.BillingForm.InputISOptionalLabel.fontSize.rawValue)
+        isOptionalLabel?.textColor = .doveGray
```

Comment:
It seems to me the only `update` is the hidden state. Everything else should be a `setup`?